locale/ca.po

```diff
@@ -266,6 +278,22 @@ msgctxt "model:ir.message,text:msg_no_session"
 msgid "Session not found."
 msgstr "Sessió no trobada."
 
+msgctxt "model:ir.message,text:msg_origin_line_with_move"
+msgid ""
+"The line have a move %(move)s related. Delete the line will create a cancell"
```

Comment:
Entenc que hauria de ser:

The line has move "%(move)s" related to it. Deleting the line will create a cancelled move. Do you want to continue?

Tot i que no ho acabo d'entendre.

Reply:
És un avís que salta quan es vol eliminar una línia d'extracte. Avisa que no s'eliminarà l'assentament associat, sinó que es crearà una contrapartida, un assentament de cancel·lació d'aquest. Així mantenim l'històric.